src/coreclr/tests/helixpublishwitharcade.proj

```diff
@@ -15,9 +15,9 @@
       <!-- This specifies what properties are needed to be passed down as global properties to a child project. -->
 
       <_PropertiesToPass>
-        TargetArchitecture=$(TargetArchitecture);
-        TargetOS=$(TargetOS);
-        Configuration=$(Configuration);
+        TargetArchitecture=$(__TargetArchitecture);
```

Comment:
Please note that TargetArchitecture and Configuration use slightly different names in <code>_PropertiesToPass</code> in send-to-helix-step - <code>__BuildArch</code> and <code>__BuildType</code>, respetively.